ext/xmlreader/php_xmlreader.stub.php

```diff
@@ -71,11 +71,11 @@ public function setParserProperty(int $property, bool $value) {}
     public function setRelaxNGSchema(?string $filename) {}
 
     /** @return bool */
-    public function setRelaxNGSchemaSource(?string $source) {}
+    public function setRelaxNGSchemaSource(?string $value) {}
```

Comment:
I think I'd keep `$source` here and in case of the next change. But let's wait for Nikita's preference too :)

Reply:
I agree that at least for the `XML()` function the name `$value` is not ideal. So yeah, keep `$source`, or possibly use `$code`.

Reply:
I just rebased on Nikita's ZPP fix and adjusted these parameter names.